services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/_resource_usage_tracker.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+from fastapi_pagination.api import create_page
+from fastapi_pagination.bases import AbstractPage
+from models_library.api_schemas_webserver.resource_usage import ContainerGet
+
+from ..models.pagination import LimitOffsetPage, LimitOffsetParamsWithDefault
+from ..models.resource_tracker_container import ContainersPage
+from ..services import resource_tracker_container_service
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter()
+
+
+@router.get(
+    "/usage/containers",
+    response_model=LimitOffsetPage[ContainerGet],
+    operation_id="list_containers",
+    description="Returns a list of tracked containers for a given user and product",
+)
+async def list_containers(
+    page_params: Annotated[LimitOffsetParamsWithDefault, Depends()],
+    containers_page: Annotated[
+        ContainersPage,
+        Depends(resource_tracker_container_service.list_containers),
+    ],
+) -> AbstractPage[ContainerGet]:
```

Comment:
I do not think you need this... whatever you return, it will be parsed by what you defined in `response_model=`

Reply:
correct will fix in upcoming PR